commerce_coordinator/apps/orders/urls.py

```diff
@@ -0,0 +1,12 @@
+"""
+Orders app URLS
+"""
+
+from django.urls import path
+
+from commerce_coordinator.apps.orders.views import get_user_orders__ecommerce
+
+app_name = 'orders'
+urlpatterns = [
+    path('ecommerce/', get_user_orders__ecommerce, name='orders_ecommerce'),
```

Comment:
I would make this path something like `orders/` or `order_history/`; this API will be the generic endpoint and won't have anything to do with legacy ecommerce once we split that backend query out into its own component/app.  May also want to remove "ecommerce" from the `name` (though we might leave it in the view name, since that view currently does rely on legacy ecommerce.)  Alternatively, these changes could be part of the work to split frontend API from backend query.

Reply:
Agreed- thanks!